tests/tools/pytest_fixtures/test_orm.py

```diff
@@ -1,19 +1,16 @@
-"""Tests for the :mod:`aiida.manage.tests.pytest_fixtures` module."""
+"""Tests for the :mod:`aiida.tools.pytest_fixtures.orm` module."""
 
 import uuid
 
 import pytest
 
-from aiida.manage.configuration import get_config
-from aiida.manage.configuration.config import Config
 from aiida.orm import Computer
 from aiida.transports import AsyncTransport, BlockingTransport
 from aiida.transports.plugins import _AsyncSSH, _OpenSSH
 
-
-def test_profile_config():
```

Comment:
This test is already covered in `tests/tools/pytest_fixtures/test_configuration.py` so I removed it here.